xmtp_mls/src/client.rs

```diff
@@ -249,6 +249,14 @@ where
         self.context.inbox_id()
     }
 
+    pub async fn find_inbox_id_from_address(
+        &self,
+        address: String,
+    ) -> Result<Option<String>, ClientError> {
+        let results = self.api_client.get_inbox_ids(vec![address.clone()]).await?;
+        Ok(results.get(&address).cloned())
```

Comment:
```suggestion
        Ok(results.remove(&address))
```

Reply:
can avoid the clone by making results mutable and using [`HashMap::remove`](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.remove) instead of `get`